deploy/charts/crate-operator/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
And same for this...

Reply:
I used `helm create <name>` to create the Chart structure and I didn't really change the template for the serviceAccount.
I can change it if you think it's a problem.

Reply:
I am ok with it if this is what helm recommends, just curious why.